terraform/modules/hub/analytics.tf

```
@@ -90,3 +90,64 @@ resource "aws_ecs_service" "analytics" {
     ]
   }
 }
+
+resource "aws_ecs_task_definition" "analytics_fargate" {
+  family                   = "${var.deployment}-analytics"
+  container_definitions    = data.template_file.analytics_task_def.rendered
+  execution_role_arn       = module.analytics_ecs_roles.execution_role_arn
+  network_mode             = "awsvpc"
+  requires_compatibilities = ["FARGATE"]
+  cpu                      = 256
+  memory                   = 512
+}
+
+resource "aws_ecs_service" "analytics_fargate" {
+  name            = "${var.deployment}-analytics"
+  cluster         = aws_ecs_cluster.fargate-ecs-cluster.id
+  task_definition = aws_ecs_task_definition.analytics_fargate.arn
+
+  desired_count                      = var.number_of_apps
+  deployment_minimum_healthy_percent = 50
+  deployment_maximum_percent         = 100
+
+  load_balancer {
+    target_group_arn = aws_lb_target_group.ingress_analytics.arn
+    container_name   = "nginx"
+    container_port   = "8443"
+  }
+
+  network_configuration {
+    subnets = aws_subnet.internal.*.id
+
+    security_groups = [
+      aws_security_group.analytics_task.id,
```

Not essential, but might be one of those things that gets forgotten about later.
```suggestion
      aws_security_group.analytics_task.id,
      aws_security_group.hub_fargate_microservice.id,
```

---

Oh, my understanding was `hub_fargate_microservice` basically meant that prometheus could connect to it

---

ah ok, yeah I guess it makes sense to add it then, just in case it gets instrumented with a metrics endpoint in future :+1: